subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3341,6 +3344,63 @@ static void le_per_adv_recv_enable(struct net_buf *buf, struct net_buf **evt)
 	ccst = hci_cmd_complete(evt, sizeof(*ccst));
 	ccst->status = status;
 }
+
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC_ADV_LIST)
+static void le_add_dev_to_pl(struct net_buf *buf, struct net_buf **evt)
```

Comment:
I think I already mentioned this, but after the whitelist was rename filter accept list, and abbreviated to `fal`, I suggest that we use the abbraviation/acronym `pal` for the **p**eriodic **a**dvertiser **l**ist.

Reply:
I see that you added another commit in this PR adding `const` to a bunch of functions (which arguably should have been a PR on its own), but to follow that trend, could/should `struct net_buf **evt` have been `struct net_buf **const evt`here as well then?